nav2-usability/scenario-code/deeplink-pathparam/router.dart

```diff
@@ -18,6 +18,40 @@ class Book {
   Book(this.title, this.author);
 }
 
+class AppState extends ChangeNotifier {
+  Book? _selectedBook;
+
+  Book? get selectedBook => _selectedBook;
+
+  set selectedBook(Book? book) {
+    _selectedBook = book;
+    notifyListeners();
+  }
+
+  final List<Book> books = [
```

Comment:
It feels weird to include this in app state, This is likely a http request response in a real app. For demo purpose, we can just define this globally somewhere.

Reply:
good suggestion - fixed